src/edge_collections.rs

```diff
@@ -0,0 +1,52 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2022
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use numpy::IntoPyArray;
+use pyo3::prelude::*;
+use pyo3::Python;
+
+/// A simple container that contains a vector representing edges in the
+/// coupling map that are found to be optimal by the swap mapper.
+#[pyclass(module = "stoachstic_swap")]
+#[pyo3(text_signature = "(/")]
```

Comment:
Yeah, I still need to go through and update all the docs and text signatures for everything. I just pushed up what I had before it was ready to start the conversation around this PR because the performance was good and it is a big discussion around the use of rust.